agent/rpc_from.go

```diff
@@ -12,15 +12,20 @@ import (
 )
 
 func (a *orbAgent) handleGroupMembership(rpc fleet.GroupMembershipRPCPayload) {
-
-	// if this is the full list, reset all group subscriptions and subscribed to this list
-	if rpc.FullList {
+	// todo think in a better approach (Create a new handle to unsubscribe)
```

Comment:
see other comments on rpc here. let's revert changes to this function and work on delete group from group channel.